app.R

```diff
@@ -100,6 +126,13 @@ ui <- dashboardPage(
       menuItem("Cam Map", tabName = "basemap", icon = icon("camera")),
       menuItem("User Inputs", tabName = "userInputs", icon = icon("user")),
       hidden(
+        selectInput(
+          "neighbourhood_names",
+          label = "Select a Neighbourhood:",
+          choices=(neighbourhood_names),
+          selected= "SURREY"
+          
```

Comment:
Let's remove the unnecessary line inside function parameters.
Also try to keep a consistent spacing before and after parameter "=".